daemon/config/config.go

```diff
@@ -47,8 +44,22 @@ const (
 	DisableNetworkBridge = "none"
 	// DefaultInitBinary is the name of the default init binary
 	DefaultInitBinary = "docker-init"
+
+	// StockRuntimeName is the reserved name/alias used to represent the
+	// OCI runtime being shipped with the docker daemon package.
+	StockRuntimeName = "runc"
+	// LinuxV1RuntimeName is the runtime used to specify runc+shimv1
+	LinuxV1RuntimeName = "com.docker.runtime.runc.v1"
+	// LinuxV2RuntimeName is the runtime used to specify runc+shimv2
+	LinuxV2RuntimeName = "com.docker.runtime.runc.v2"
```

Comment:
Should we create a `type` for this, so that they are more of an enum (and nicely grouped together in the Go Docs, ah-la https://godoc.org/github.com/docker/docker/api/types/mount#Consistency

Reply:
Tried this, but it is painful to deal with. Opted to not do this.